source/Octopus.Tentacle.Tests.Integration/Support/TentacleConfigurationsAttribute.cs

```diff
@@ -18,13 +20,28 @@ public class TentacleConfigurationsAttribute : TentacleTestCaseSourceAttribute
             : base(
                 typeof(TentacleConfigurationTestCases),
                 nameof(TentacleConfigurationTestCases.GetEnumerator),
-                new object[] {testCommonVersions, testCapabilitiesServiceVersions, testNoCapabilitiesServiceVersions, testScriptIsolationLevelVersions, testDefaultTentacleRuntimeOnly, additionalParameterTypes})
+                new object[] { testCommonVersions, testCapabilitiesServiceVersions, testNoCapabilitiesServiceVersions, testScriptIsolationLevelVersions, testDefaultTentacleRuntimeOnly, additionalParameterTypes })
         {
         }
     }
 
     static class TentacleConfigurationTestCases
     {
+        static readonly Type[] ScriptServiceV2Types = { typeof(IClientScriptServiceV2), typeof(IAsyncClientScriptServiceV2) };
+        static readonly Type[] ScriptServiceV1Types = { typeof(IClientScriptService), typeof(IAsyncClientScriptService) };
+
+        static readonly Type[] CurrentScriptServiceTypes = ScriptServiceV2Types;
```

Comment:
Can't have a null dictionary key (as the current version is `null`, so just have a preset value for the current version